### PR TITLE
Revert "Update versions and references for 0.0.3 publishing"

### PR DESCRIPTION
**Pull request by tinkrtailor**
Reverts projectawakening/world-chain-contracts#85

### CODE REVIEW

File: end-to-end-tests/package.json [hunk 1]
```diff
@@ -47,9 +47,9 @@
     "@latticexyz/world": "2.0.0-transaction-context-98ef570f",
     "@latticexyz/world-modules": "2.0.0-transaction-context-98ef570f",
     "@latticexyz/gas-report": "2.0.0-transaction-context-98ef570f",
-    "@eveworld/smart-object-framework": "link:../mud-contracts/smart-object-framework",
-    "@eveworld/world": "link:../mud-contracts/world",
-    "@eveworld/common-constants": "link:../mud-contracts/common/constants"
+    "@eveworld/smart-object-framework": "0.0.1",
```
Comment (0xxlegolas) [inline end-to-end-tests/package.json:50]:
```suggestion
    "@eveworld/smart-object-framework": "0.0.2",
```

**Review by MerkleBoy — APPROVED** [review]
lgtm 👍

**Review by fc-tezza — APPROVED** [review]

**Comment by 0xxlegolas** [comment]
can we make it `"0.0.2` for all ?

File: mud-contracts/core/package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     "@latticexyz/store": "2.0.0-transaction-context-98ef570f",
     "@latticexyz/world": "2.0.0-transaction-context-98ef570f",
     "@latticexyz/world-modules": "2.0.0-transaction-context-98ef570f",
-    "@eveworld/common-constants": "0.0.3"
+    "@eveworld/common-constants": "0.0.1"
```
Comment (gunnigylfa) [inline mud-contracts/core/package.json:28]:
```suggestion
    "@eveworld/common-constants": "0.0.2"
```

File: mud-contracts/world/package.json [hunk 3]
```diff
@@ -24,8 +24,8 @@
     "@latticexyz/world": "2.0.0-transaction-context-98ef570f",
     "@latticexyz/world-modules": "2.0.0-transaction-context-98ef570f",
     "@latticexyz/gas-report": "2.0.0-transaction-context-98ef570f",
-    "@eveworld/smart-object-framework": "0.0.3",
-    "@eveworld/common-constants": "0.0.3"
+    "@eveworld/smart-object-framework": "0.0.1",
```
Comment (gunnigylfa) [inline mud-contracts/world/package.json:27]:
```suggestion
    "@eveworld/smart-object-framework": "0.0.2",
```

File: mud-contracts/world/package.json [hunk 4]
```diff
@@ -24,8 +24,8 @@
...
+    "@eveworld/common-constants": "0.0.1"
```
Comment (gunnigylfa) [inline mud-contracts/world/package.json:28]:
```suggestion
    "@eveworld/common-constants": "0.0.2"
```

File: end-to-end-tests/package.json [hunk 5]
```diff
@@ -47,9 +47,9 @@
     "@latticexyz/world": "2.0.0-transaction-context-98ef570f",
     "@latticexyz/world-modules": "2.0.0-transaction-context-98ef570f",
     "@latticexyz/gas-report": "2.0.0-transaction-context-98ef570f",
-    "@eveworld/smart-object-framework": "link:../mud-contracts/smart-object-framework",
-    "@eveworld/world": "link:../mud-contracts/world",
-    "@eveworld/common-constants": "link:../mud-contracts/common/constants"
+    "@eveworld/smart-object-framework": "0.0.2",
+    "@eveworld/world": "0.0.1",
```
Comment (gunnigylfa) [inline end-to-end-tests/package.json:51]:
```suggestion
    "@eveworld/world": "link:../mud-contracts/world",
```

File: end-to-end-tests/package.json [hunk 6]
```diff
@@ -47,9 +47,9 @@
...
+    "@eveworld/common-constants": "0.0.1"
```
Comment (gunnigylfa) [inline end-to-end-tests/package.json:52]:
```suggestion
    "@eveworld/common-constants": "link:../mud-contracts/common/constants"
```

File: standard-contracts/package.json [hunk 7]
```diff
@@ -28,8 +28,8 @@
     "@latticexyz/schema-type": "2.0.0-transaction-context-98ef570f",
     "@latticexyz/world": "2.0.0-transaction-context-98ef570f",
     "@latticexyz/store": "2.0.0-transaction-context-98ef570f",
-    "@eveworld/world": "0.0.3",
-    "@eveworld/common-constants": "0.0.3"
+    "@eveworld/world": "0.0.1",
```
Comment (gunnigylfa) [inline standard-contracts/package.json:31]:
```suggestion
    "@eveworld/world": "0.0.2",
```

File: standard-contracts/package.json [hunk 8]
```diff
@@ -28,8 +28,8 @@
...
+    "@eveworld/common-constants": "0.0.1"
```
Comment (gunnigylfa) [inline standard-contracts/package.json:32]:
```suggestion
    "@eveworld/common-constants": "0.0.2"
```

**Pull request merged**